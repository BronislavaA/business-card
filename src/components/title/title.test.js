import React from 'react';
import renderer from 'react-test-renderer';
import Title from './title'

test('renders Title component', () => {
    const component = renderer.create(
      <Title />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });