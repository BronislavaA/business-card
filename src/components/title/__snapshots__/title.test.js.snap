// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title component 1`] = `
<div
  className="sc-bdVaJa kxiiak"
>
  <h1>
    Evan Bradley
  </h1>
  <p>
    Climate Change Analyst
  </p>
</div>
`;
